pages/common/carp.md

```diff
@@ -0,0 +1,28 @@
+# carp
+
+> REPL and build tool for Carp.
+> More information: <https://carp-lang.github.io/carp-docs/Manual.html>.
+
+- Start a REPL:
+
+`carp`
+
+- Start a REPL with a custom prompt:
+
+`carp --prompt "{{> }}"`
+
+- Build a `.carp` file:
```

Comment:
Just read this over and realized it could be interpreted as creating ./.carp (a hidden files)

I forgot what we do for showing extensions but removing the `.` before carp removes any possible confusion.

This is a petty change and I don't mind if you disagree and resolve.

Reply:
It is funny that you weren't involved in #6455

Reply:
Scary hidden files 😁